domain.yml

```diff
@@ -169,6 +195,8 @@ templates:
     - text: "I can't sign you up if you don't provide your email 😉"
   utter_ask_isbot:
     - text: "Yep I'm a bot!"
+  utter_cant_answer_tech_question:
+    - text: "I can't answer that"
```

Comment:
I think we should change this to "I can't answer that, please take a look at our docs to find your answer: (link)", but we can change that in the platform rather than here